src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -120,9 +118,12 @@ private void WriteClientMethodBody(CodeWriter writer, LowLevelClientMethod clien
             writer.Line();
         }
 
+        private static readonly CSharpType RequestOptionsParameterType = new CSharpType(typeof(RequestOptions), true);
+        private static readonly Parameter RequestOptionsParameter = new Parameter("options", "The request options", RequestOptionsParameterType, Constant.Default(RequestOptionsParameterType), false);
+
         private void WriteClientMethodDecleration(CodeWriter writer, LowLevelClientMethod clientMethod, bool async)
         {
-            var parameters = clientMethod.Parameters;
+            var parameters = clientMethod.Parameters.Concat(new Parameter[] { RequestOptionsParameter });
```

Comment:
`.Append(RequestOptionsParameter)`

Reply:
Wow! Somehow I've gone this long not knowing `Append` exits!  Thanks for the pointer and saving me a bunch of characters down the road.